DNN Platform/Library/Entities/Modules/Actions/IModuleEventHandler.cs

```diff
@@ -12,6 +12,8 @@ public interface IModuleEventHandler
 
         void ModuleRemoved(object sender, ModuleEventArgs args);
 
-        void ModuleDeleted(object sender, ModuleEventArgs args);
+        void ModuleDeleted(object sender, ModuleEventArgs args);
+
+        void ModuleGenericEvent(object sender, ModuleGenericEventArgs args);
```

Comment:
Adding a new member to an interface (that we expect third parties to implement) is a breaking change, unfortunately…